src/doc/book/unsized-types.md

```diff
@@ -47,14 +47,14 @@ pointers, can use this `impl`.
 # ?Sized
 
 If you want to write a function that accepts a dynamically sized type, you
-can use the special bound, `?Sized`:
+can use the special syntax, `?Sized`:
```

Comment:
Does `?Sized` remove the `Sized` trait, or adds a "not `Sized`" trait in addition to the implicit `Sized` trait?